contrib/hyperscan/matching/input_matchers/test/config_test.cc

```diff
@@ -0,0 +1,154 @@
+#include <utility>
+#include <vector>
+
+#include "test/mocks/server/factory_context.h"
+
+#include "contrib/hyperscan/matching/input_matchers/source/config.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Matching {
+namespace InputMatchers {
+namespace Hyperscan {
+
+const std::string& yamlString() {
+  CONSTRUCT_ON_FIRST_USE(std::string, R"EOF(
```

Comment:
sorry -- this macro is overkill for individual constant strings. Instead just use `constexpr absl::string_view`.

it's more appropriate for the vector of strings you also create.

Reply:
Ok, I fixed them.